src/ViewInstance.js

```diff
@@ -289,10 +293,44 @@ class ViewInstance extends React.Component {
     this.setState({ isImportRecordModalOpened: false });
   }
 
+  toggleCopyrightModal = () => {
+    this.setState(prevState => ({ isCopyrightModalOpened: !prevState.isCopyrightModalOpened }));
+  };
+
   toggleFindInstancePlugin = () => {
     this.setState(prevState => ({ findInstancePluginOpened: !prevState.findInstancePluginOpened }));
   };
 
+  // Get all identifiers for all records
+  getIdentifiers = (data) => {
+    const { identifierTypesById } = data;
+    const { ISBN, ISSN } = indentifierTypeNames;
+    const records = this.props?.resources?.selectedInstance?.records;
+    if (!records) {
+      return null;
+    }
+    const allIdentifiers = [].concat(
+      records.map(record => record.identifiers)
+    )[0];
```

Comment:
@AndrewIsh it looks like only first record in the array is being used here. What if the first record has no identifiers at all? Sorry for this question but I'm having a hard time understanding what `getIdentifiers` does. The comment says `Get all identifiers for all records` but the code doesn't seem to do that. Could you please clarify what is happening here?

Reply:
> I think selectedInstance will only return a single record right
That's my understanding. Whereas `records` is an array but only ever seems to contain a single element, but I'm assuming it can contain more.